scripts/verify-spelling.sh

```diff
@@ -15,14 +15,14 @@
 # limitations under the License.
 ##########
 # This script verifies mispellings in location. Today it only supports
-# verifying English locale but can be modified in a future to support 
+# verifying English locale but can be modified in a future to support
 # also other locales.
 # You need to run this script inside the root directory of "website" git repo.
```

Comment:
Is this intended to convey that the user needs to run this script inside their local clone of  the kubernetes/website git repo?

Reply:
yes